examples/decoding/plot_decoding_xdawn_eeg.py

```diff
@@ -62,30 +62,29 @@
                 add_eeg_ref=False, verbose=False)
 
 # Create classification pipeline
-clf = make_pipeline(Xdawn(n_components=3),
-                    EpochsVectorizer(),
+X = EpochsVectorizer().fit_transform(epochs)
+labels = epochs.events[:, -1]
+clf = make_pipeline(XdawnTransformer(n_components=3,
+                    n_chan=epochs.info['nchan']),
                     MinMaxScaler(),
                     LogisticRegression(penalty='l1'))
```

Comment:
The `Vectorizer` is in the other PR. This was working because `XdawnTransformer` `transform` method was a 2D matrix.
I changed it back to original and added a new example showcasing the `XdawnTransformer`. However it utilizes `Vectorizer` so it won't work now.